docs/proposals/VIP-2.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: default
+title: [VIP-2] Removing Per Record Offset Metadata From Venice-Server Storage With Heartbeats
+parent: Community Guides
+permalink: /docs/proposals/VIP_TEMPLATE.md
```

Comment:
For generating a reachable URL:

```suggestion
permalink: /docs/proposals/vip-2
```

Reply:
Please also update the Proposals [table](https://github.com/ZacAttack/venice/blob/VIP-2/docs/proposals/proposals.md).